wolfcrypt/src/port/maxim/maxq10xx.c

```diff
@@ -106,16 +106,14 @@ static int tls13_server_key_len           = -1;
 
 #endif /* WOLFSSL_MAXQ108X */
 
+#define USE_MAXQ_TEST_KEY
+
+#ifdef USE_MAXQ10XX_TEST_KEY
```

Comment:
This still requires a code change in a file tracked upstream. My goal here was to use the existing test by default and at build-time allow someone a way to supply their own without modifying code.